packages/web/src/scss/components/Collapse/_Collapse.scss

```diff
@@ -17,6 +30,7 @@
                     /* stylelint-disable declaration-no-important -- This Important rule is important here because it overrides inline styles applied by JavaScript */
```

Comment:
Tip: For single-line tweaks, you can use `stylelint-disable-next-line` which saves from having to write `stylelint-enable`.

Reply:
Ok, thanks for tip, fixed.